python/ml4ir/applications/ranking/tests/test_cyclic_lr_schedule.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+import warnings
+from ml4ir.base.features.preprocessing import *
+from ml4ir.base.model.optimizers import cyclic_learning_rate
+
+warnings.filterwarnings("ignore")
+
+INPUT_DIR = "/ml4ir/applications/ranking/tests/data/"
+KEEP_ADDITIONAL_INFO = 0
+NON_ZERO_FEATURES_ONLY = 0
+
+
+class TestRanklibConversion(unittest.TestCase):
```

Comment:
@mohazahran You should rename this test class and remove the additional constants described above.

Reply:
damn copy and paste :)